doc/api/dgram.md

```diff
@@ -35,8 +35,9 @@ server.bind(41234);
 added: v0.1.99
 -->
 
-The `dgram.Socket` object is an [`EventEmitter`][] that encapsulates the
-datagram functionality.
+* Extends: {EventEmitter}
+
+Encapsulates the datagram functionality.
```

Comment:
Is this sentence meaningful/significant in context? Or should it be removed?

Reply:
(Non-blocking question.)

Reply:
dgram.Socket is self-explanatory, and I think this sentence should be removed.
I'd retained it in original PR, as it isn't related to class extension.